tests/CMakeLists.txt

```diff
@@ -196,6 +196,11 @@ add_test(mont mont)
 
 # ocsp_test
 if(ENABLE_EXTRATESTS)
```

Comment:
What about defining this as relative to CMAKE_CURRENT_SOURCE_DIR instead, so we don't assume that there are any certs in the install directory at all at test-time.